### DISCUSSION TITLE
Query on pytorch version

### DISCUSSION THREAD

**Discussion by ssuhas76** [mrdbourke/pytorch-deep-learning#1108] [General]
I just bought this course on Udemy and started learning. Had a question about the pytorch version used for teaching on the course. It say 1.10.xxx but currently the official version of pytorch is 2.4. 

Is it Ok to learn with old version or will this course be updated to the new version?

**Comment by mrdbourke** [comment]
Hey there,Great question. The course does mostly use PyTorch 1.10.XXX (the version that was out when I started making it). However, any pieces which have required updates since launch have been updated (I constantly update the parts of the course that need changing).Most of the materials are the fundamentals which haven’t changed in several versions. If you do run into any errors please let me know and I’ll get them fixed :) Best,Daniel BourkeOn 6 Oct 2024, at 5:23 AM, Suhas S ***@***.***> wrote:﻿
I just bought this course on Udemy and started learning. Had a question about the pytorch version used for teaching on the course. It say 1.10.xxx but currently the official version of pytorch is 2.4.
Is it Ok to learn with old version or will this course be updated to the new version?

**Comment by ssuhas76** [comment]
That's perfect!! Thanks @mrdbourke
